features/exception_handling/developer_is_told_which_scenario_fataled.feature

```diff
@@ -0,0 +1,45 @@
+@isolated
+Feature: As a Developer
```

Comment:
This needs a scenario covering what happens when an error occurs outside of a spec execution (e.g. syntax error in the spec so it fails at the load stage)